cmd/kpod/create_cli.go

```diff
@@ -40,6 +42,81 @@ func addWarning(warnings []string, msg string) []string {
 	return append(warnings, msg)
 }
 
+func parseVolumes(volumes []string) error {
+	for _, volume := range volumes {
+		arr := strings.Split(volume, ":")
+		switch len(arr) {
+		case 2:
+			if err := validateVolumeHostDir(arr[0]); err != nil {
```

Comment:
Do these two checks outside of the switch block,  I would remove switch block and 
verify arr[0]
verify arr[1]
If len > 2 
veriry arr[2]

Reply:
fixed